stable/elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 1.1.3
+version: 1.1.4
```

Comment:
Can you bump minor version instead of patch because it is a new feature?

Reply:
ok, will do once the above point is clarified.

Reply:
done